webui/src/pages/sidepanel/navigation-item.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Add license header comment.

This file should go to `src/components` rather than `src/pages`, right?

Reply:
This is a very sidepanel specific component so I'd say we can leave it here.